ledger/metrics/collection/reset-all.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+# Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+
+docker-compose down -v 
+
+rm -rf graphite/conf/*
+rm -rf graphite/statsd_conf/*
```

Comment:
Can we use Docker volumes for these?

Reply:
We could. Currently, I perceive that it would be more complicated to inspect the config, if it is stored in a volume. Which benefit do you expect from using volumes?

Reply:
Just makes it easier to tear them down. But I don't mind too much either way.

Reply:
Ok, resolving.